cuda/matrix/ell_kernels.cu

```diff
@@ -212,7 +214,7 @@ void spmv(std::shared_ptr<const CudaExecutor> exec,
     const int num_worker_per_row = std::get<2>(data);
 
     /**
-     * info is the parameter for selecting the cuda kernel.
+     * info is the parameter for selecting the device kernel.
```

Comment:
The intention of these changes is to make the files as identical as possible, so the other PR can consist (almost) entirely of automated changes. Otherwise it would be impossible to review